src/verify/fragments/document-status/obligationRecords/obligationRecordVerifier.types.ts

```
@@ -0,0 +1,31 @@
+  };
+  reason?: ObligationRecordsErrorReason;
+};
+
+export type ObligationRecordsErrorFragment = Omit<ErrorVerificationFragment<never>, 'data'> & {
+  data?: never;
+  reason: ObligationRecordsErrorReason;
+};
+
+export type ObligationRecordsVerificationFragment =
+  | ObligationRecordsResultFragment
+  | ObligationRecordsErrorFragment;
```

ESLint

[error] Replace `|·ObligationRecordsResultFragment⏎·` with `ObligationRecordsResultFragment`

(prettier/prettier)